### ISSUE TITLE
Disable attribute is not removed when it's 'false' which causes some inconveniences in a11y

### ISSUE THREAD

**Issue by elizavetaKrainova** [telekom/scale#2390]
**Scale Version**
3.0.0-beta.153

**Framework and version**
Vue 3.5.12
Are you using one of the auto-generated scale wrapper packages? 
Not sure

**Current Behavior**
When the `disabled `prop is set to `false`, it is not removed from the DOM. As a result, screen readers announce the button as _unavailable_, even though it is actually available and active.

**Expected Behavior**
When the `disabled `prop is set to false, it should be removed from the DOM so that screen readers recognize the button as enabled.

**Code Reproduction**
```
<scale-button
	class="scale-btn"
	type="button"
	:variant="primary ? 'primary' : 'secondary'"
	:size="big ? '' : 'small'"
	@click="handleClick"
	:disabled="disabled"
	:style="darkMode ? darkStyle : ''"
>
</scale-button> 
```

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Version: 136.0.7103.93

**Additional context**
Was trying to test application with NVDA screen reader. Without `disabled` prop everything work as it supposed to, button is clickable with enter button. With `disabled` prop button becomes unavailable for screen reader.

**Comment by amir-ba** [comment]
Ok. Thank you , I now understand the issue I thought it had to do with the attribute itself, but in this case the best way to solve this is not to show the attribute in the html part at all to not confuse the screen reader . We will try to change this as soon as possible.
